GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -136,17 +126,81 @@ public void runImport(File file, String serverBase, Map<String, String> criteria
         Map<Long, Long> columnIndexToQuestionId = processHeader(sheet);
```

Comment:
As you pointed out there will be issues processing the header row of  `GEO` questions.  These will fail because of the split columns.